CHANGELOG.txt

```diff
@@ -246,6 +253,13 @@
                     - Object.entries()
                     - Object.getPrototypeOf()
                 * Drop creme.widget.template(); Use String.prototype.template instead.
+                * Refactor of creme.ajax.Backend:
+                    - Drop jquery.forms plugin; Use FormData standard API instead.
+                    - $.validateHTML5() is removed (came along jquery.forms)
+                    - Add support for download & upload progress event : new 'uploadProgress' & 'progress' options.
+                        - Refactor 'ListViewController' & 'creme.bricks.BricksReloader'
+                        - 'creme.ajax.Query' now accepts new 'uploadProgress' & 'progress' options.
```

Comment:
Not a breaking change?

Reply:
The uploadProgress/progress handler part is non breaking (new feature).